zombienet/configs/moonbeam-polkadot.toml

```diff
@@ -0,0 +1,46 @@
+[settings]
+provider = "native"
+
+[relaychain]
+default_command = "polkadot"
+default_args = [
+    "-lruntime::bridge-grandpa=trace"
+]
+chain_spec_path = "zombienet/specs/polkadot-local.json"
+
+[[relaychain.nodes]]
+name = "alice"
```

Comment:
nit: Would be nice to have a different naming convention for nodes since alice/alith are often used for accounts.